whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -48,15 +48,59 @@ <h4 id='error_tag' class='light warning'></h4>
         {{ vendor_form.story }}
         <p class=" field_text">In Port:</p>
         {{ vendor_form.status }}
-        <h3>Location</h3>
-        <p class="field_text">Street Address*:</p>
+        <h3>Location</h3><br>
+
+    <!-- Code for Map based location inputs start -->
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I suppose it might be a good idea to have this tag in the document, but it should be in base.html if it's in there at all.